src/openrct2/rct12/RCT12.h

```diff
@@ -651,31 +651,31 @@ enum class RCT12PeepType : uint8_t
     Invalid = 0xFF
 };
 
-struct RCT12SpriteBase
+struct RCT12EntityBase
 {
-    RCT12SpriteIdentifier sprite_identifier;           // 0x00
+    RCT12EntityIdentifier EntityIdentifier;            // 0x00
     uint8_t type;                                      // 0x01
     uint16_t next_in_quadrant;                         // 0x02
     uint16_t next;                                     // 0x04
     uint16_t previous;                                 // 0x06
     RCT12EntityLinkListOffset linked_list_type_offset; // 0x08
-    uint8_t sprite_height_negative;                    // 0x09
-    uint16_t sprite_index;                             // 0x0A
+    uint8_t EntityHeightNegative;                      // 0x09
```

Comment:
The EntityHeightNegative, EntityWidth and EntityHeightPositive are also related to the sprite. Its used for calculating the bounding box for invalidation and is based on the max size of the sprite that is drawn. Up for debate if that means its Entity or Sprite.